File: src/luogu/questions/p5015/Main.java
package luogu.questions.p5015;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String s = sc.nextLine();
        int i = 0;
        for (char c : s.toCharArray()) {
            if (c == ' ' || c == '\n')
                continue;
            i++;
        }
        System.out.println(i);
    }
}

File: src/luogu/questions/p5015/Main.cpp
// https://www.luogu.com.cn/problem/P5015
#include <bits/stdc++.h>

using namespace std;

int main() {
  string s;
  int i = 0;
  while (cin >> s) {
    i += s.length();
  }
  cout << i << endl;
  return 0;
}
